src/main/java/Duke.java

```diff
@@ -2,30 +2,109 @@
 
 public class Duke {
 
-    public static void main(String[] args) {
-        String command;
-        Scanner in = new Scanner(System.in);
+    static String by;
+    static String at;
+    static Task[] tasks = new Task[100];
+    static String[] keyword = new String[100];
+    static int numberOfTasks = 0;
+
+    public static void printBye() {
+        System.out.println("____________________________________________________________");
```

Comment:
to improve readability, i suggest to have a section divider variable for this! (: